### ISSUE TITLE
Add an option to still prevent screen-saver prevention when video is not playing

### ISSUE THREAD

**Issue by HavKT** [ram-on/SkyTube#478]
I was fine when I used 2.95 but this update only makes it worse as I have to continuously click when the video is not playing plus I have to reenter my credentials when the phone goes off. Please don't be YouTube and provide us an option where this can be toggled.

**Comment by gzsombor** [comment]
I would never though, that this bug was a feature for someone.
 Why do you need this feature, why don't you just remove the lock screen or extend the time, until the display goes to sleep? How do you use SkyTube, so you don't see the end of the video, only later, when the screen is already locked?
 It's not YouTube, it's open source, so anyone can implement this feature :wink:

**Comment by atomjack** [comment]
Yeah I'm very confused about this. @HavKT you say you have to "continuously click when the video is not playing", is that to make sure the phone doesn't go to sleep? If that's the case, that has nothing to do with the app, but with your phone's display settings.

@gzsombor I suggest you close this if we don't get an answer soon (I am sure you agree with me that it'd be nice to pare the number of open issues;)

**Comment by HavKT** [comment]
Sorry for not responding about 2 months. @atomjack, I prefer to not go to display settings and change the time for the screen to shut off. What I am asking is an option to keep your display on like in 2.95 where it [was] considered a bug but for me it was a feature.

**Comment by HavKT** [comment]
And label this as "enhancement" since I am already aware of Display Settings and this request specifically targets when you pause the video. Anything outside of the video being watched is controlled by the screen shutoff timer. My request, as I said, is to target a toggle as to whether or not the video paused should stay on or be controlled by the screen shutoff timer. Hope you all understand.

**Comment by MyWay** [comment]
+1 for this
